projects/limitless-exchange/index.js

```diff
@@ -0,0 +1,22 @@
+const eventAbi = "event FixedProductMarketMakerCreation(address indexed creator, address fixedProductMarketMaker, address indexed conditionalTokens, address indexed collateralToken, bytes32[] conditionIds, uint256 fee)";
+const config = [
+  { factory: "0x8E50578ACa3C5E2Ef5ed2aA4bd66429B5e44C16E", resolver: "0x15A61459d65D89A25a9e91e0dc9FC69598791505", fromBlock: "13547870" },
+  { factory: "0xc397D5d70cb3B56B26dd5C2824d49a96c4dabF50", resolver: "0xc9c98965297bc527861c898329ee280632b76e18", fromBlock: "13547845" },
+];
+
+const getMarkets = async ({ factory, resolver, fromBlock }, api) => {
+  const logs = await api.getLogs({ target: factory, eventAbi, fromBlock, toTimestamp: api.timestamp });
```

Comment:
for now, lets stick with `getLogs` helper in this repo, it automatically caches the log response.

And since all log calls go through the same function, any change we need to do, we can do it in one place.